Piwik.Tracker/PiwikTracker.cs

```diff
@@ -1573,6 +1573,7 @@ private string GetRequest(int idSite)
                 (!_ecommerceLastOrderTimestamp.Equals(DateTimeOffset.MinValue) ? "&_ects=" + FormatTimestamp(_ecommerceLastOrderTimestamp) : "") +
 
                 // Various important attributes
+                // todo _customData is never assigned!

```

Comment:
Probably the same reason as for `_currentVisitTs`.

I have created https://github.com/piwik/piwik-dotnet-tracker/issues/38